themes/minimal/layouts/partials/header.html

```diff
@@ -1,18 +1,26 @@
 <header>
-  {{ if ne "" .Site.Title }}
-  <h1><a href="{{ relURL "/" }}">{{ .Site.Title }}</a></h1>
-  {{ end }}
+    {{ if ne "" .Site.Title }}
+        <h1><a href="{{ relURL "/" }}">{{ .Site.Title }}</a></h1>
+    {{ end }}
 
-  {{ if ne "" .Site.Params.Logo }}
-  <img src="{{ relURL .Site.Params.Logo }}" alt="Logo" style="margin-bottom:20px"/>
-  {{ end }}
+    {{ if ne "" .Site.Params.Logo }}
+        <img src="{{ relURL .Site.Params.Logo }}" alt="Logo" style="margin-bottom:20px"/>
+    {{ end }}
 
-  <p>{{ .Site.Params.Description }}</p>
+    <p>{{ .Site.Params.Description }}</p>
 
-  <ul class="downloads">
-    <li><a href="https://www.meetup.com/Go-London-User-Group/"><i class="fab fa-meetup" aria-hidden="true"></i></a></li>
-    <li><a href="https://www.youtube.com/c/LondonGophers"><i class="fab fa-youtube" aria-hidden="true"></i></a></li>
-    <li><a href="https://twitter.com/LondonGophers"><i class="fab fa-twitter" aria-hidden="true"></i></a></li>
-    <li><a href="https://github.com/go-london-user-group"><i class="fab fa-github" aria-hidden="true"></i></a></li>
-  </ul>
+    <ul class="downloads">
```

Comment:
I can't help thinking this would look much better with the sponsors below the blurb in the main section.

Here's a mock up moving bits around in Chrome:

![main_section](https://user-images.githubusercontent.com/3374574/61762713-be83a700-ad87-11e9-8d8e-b59c32789993.png)